src/camera.c

```diff
@@ -350,30 +349,30 @@ void disable_cutscene_camera(void) {
  * If the number passed is within 1-4, then the stack cap is set to
  * how many active viewports there are.
  */
-s32 set_active_viewports_and_max(s32 num) {
-    if (num >= 0 && num < 4) {
-        gNumberOfViewports = num;
+s32 set_active_viewports_and_max(s32 layoutID) {
```

Comment:
suggestion: Use the `ViewportCount` enum as a typehint for layoutID here if possible